src/scss/fields.scss

```diff
@@ -1,98 +1,63 @@
-@include oFormsClass(o-forms__field-group) {
+@if not $o-forms-is-silent {
+.o-forms-group {
```

Comment:
I chose this class randomly, I think it's still interesting to discuss if partial silent mode support is a good idea for Origami.